### PR TITLE
PR: Create dependabot.yml file

### PR DESCRIPTION
**Pull request by nelsonic**
See: https://github.com/dwyl/learn-tdd/issues/146

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: mix
```
Comment (SimonLab) [inline .github/dependabot.yml:3]: I haven't read the doc for dependabot, but as this repo doesn't contain any `mix`/elixir code I would think the package-ecosystem is not right here

**Review by SimonLab — COMMENTED** [review]
Github can create a config file commits for updating dependency using dependabot:
image

```yml
# To get started with Dependabot version updates, you'll need to specify which
# package ecosystems to update and where the package manifests are located.
# Please see the documentation for all configuration options:
# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates

version: 2
updates:
  - package-ecosystem: "" # See documentation for possible values
    directory: "/" # Location of package manifests
    schedule:
      interval: "daily"
```

File: .github/dependabot.yml (hunk 1 above)
Comment (SimonLab) [reply .github/dependabot.yml:3]: I think the value should be `package-ecosystem: npm`

Comment (SimonLab) [reply .github/dependabot.yml:3]: I think the value should be `package-ecosystem: npm`

Comment (nelsonic) [reply .github/dependabot.yml:3]:
```suggestion
- package-ecosystem: npm
```

**Pull request merged**